client/templates/userprofile/settings.html

```diff
@@ -1,134 +1,62 @@
 <template name="profileSettings">
-<div class="panel">
-	<div class="panel-heading">
-		<h1 class="userprofile-main-title">
-	          {{ mf 'userProfile-mainHeading' 'Profile Settings'}}
-	    </h1>
-	    <a href="" class="float-right">{{ mf 'userProfile-account' 'Account (to be added)' }} </a>
-	    <a href="" class="float-right">{{ mf 'userProfile-email' 'Email (to be added)' }}</a>
-	    <hr>
-	</div>
-    <div class="panel-body">
-		<h2 class="userprofile-sub-title">
-				{{ mf 'userProfile-subheading-basics' 'Basics'}}
-		</h2>
-		<div class="form-group {{#if realerror}}has-error{{/if}}">
-            <p class="profile_errors">{{realerror}}</p>
-                <label for="Real Name">
-                    {{ mf 'userProfile-realname-label' 'Real Name' }}
-                    </label>
-                <input
-                       type="text"
-                       class="form-control"
-                       id="realName"
-                       value="{{profile.realname}}">
+    <div class="panel">
+        <div class="panel-heading">
+            <h1 class="userprofile-main-title">
+                {{ mf 'userProfile-mainHeading' 'Profile Settings'}}
+            </h1>
+            <a href="" class="float-right">{{ mf 'userProfile-account' 'Account (to be added)' }} </a>
```

Comment:
Remove any unused UI elements. We can open a feature request for these features.